repair_type_refurbish/models/repair.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 ForgeFlow S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+
+from odoo import api, fields, models
+
+
+class Repair(models.Model):
+    _inherit = "repair.order"
+
+    repair_type_id = fields.Many2one(comodel_name="repair.type")
```

Comment:
You don't need to readd this field [again](https://github.com/OCA/repair/blob/15.0/repair_type/models/repair.py#L10)